src/derivatives/atlas.md

```diff
@@ -337,18 +333,19 @@ A guide for using macros can be found at
    "colin27-pipeline": {
       "sub-01": {
          "anat": {
-            "sub-01_space-MNI305_atlas-AAL_dseg.json": "",
-            "sub-01_space-MNI305_atlas-AAL_dseg.nii.gz": "",
-            "sub-01_space-MNI305_atlas-AAL_dseg.tsv": "",
-            "sub-01_space-MNI305_atlas-AAL_probseg.nii.gz": "",
-            "sub-01_space-MNI305_label-brain_mask.nii.gz": "",
-            "sub-01_space-MNI305_label-head_mask.nii.gz": "",
-            "sub-01_space-MNI305_T1w.nii.gz": "",
-            "sub-01_space-MNI305_T1w.json": "",
-            "sub-01_space-T1w_label-brain_mask.nii.gz": "",
-            "sub-01_space-T1w_label-head_mask.nii.gz": "",
-            "sub-01_space-T1w_T1w.nii.gz": "",
-            "sub-01_space-T1w_T1w.json": "",
+            "atlas-AAL_space-MNI305_dseg.json": "",
+            "atlas-AAL_space-MNI305_dseg.nii.gz": "",
+            "atlas-AAL_space-MNI305_dseg.tsv": "",
+            "atlas-AAL_space-MNI305_probseg.nii.gz": "",
+            "atlas-AAL_space-MNI305_seg-brain_mask.nii.gz": "",
+            "atlas-AAL_space-MNI305_seg-head_mask.nii.gz": "",
+            "atlas-AAL_space-colin27_dseg.json": "",
+            "atlas-AAL_space-colin27_dseg.nii.gz": "",
+            "atlas-AAL_space-colin27_dseg.tsv": "",
+            "atlas-AAL_space-colin27_probseg.nii.gz": "",
+            "atlas-AAL_space-colin27_seg-brain_mask.nii.gz": "",
+            "atlas-AAL_space-colin27_seg-head_mask.nii.gz": "",
+            "...", "",
```

Comment:
You are losing here the notion that you are still in subject space. Definitely not a use case for atlas.

Please roll back.